### PR TITLE
saas_client security

### PR DESCRIPTION
**Pull request by yelizariev**
This PR is an implementation of the idea of flexible configuration for Saas Client database (see #51 and new section "Client database customization" at root Readme file).

The idea is to have separate addons for customizing security rights on Client Databases. It makes customization flexible from one side, and easier to maintain such addons from other side.

[Maximum allowed users feature](https://github.com/yelizariev/odoo-saas-tools/blob/e0ca4d780ad44cb8203ded584073b3949d0cdee9/saas_client/models/res_user.py#L32-L40) could be moved to a separate addon too. For now, I've removed code that seems doesn't work after a lot of update and refactoring we made.

cc @kaerdsar , @dhbahr , @tarzan0820

### CODE REVIEW

File: saas_client/security/groups.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="group_no_client" model="res.groups">
```
Comment (saltonmassally) [inline saas_client/security/groups.xml]: Can we rename this group_saas_support and call maybe Tech Support or Saas Support?

**Comment by saltonmassally** [comment]
Tiny cosmetic change but :+1:

File: saas_client/security/groups.xml (hunk 1 above)
Comment (yelizariev) [inline saas_client/security/groups.xml]: good idea

**Comment by saltonmassally** [comment]
@yelizariev since we have not yet heard from @kaerdsar & @dhbahr  should we go ahead and merge this PR into 8.0?

**Comment by yelizariev** [comment]
ok, I've merged it

**Pull request merged**